edalize/templates/vivado/vivado-run.tcl.j2

```diff
@@ -24,3 +27,7 @@ if { [get_property PROGRESS [get_runs impl_1]] != "100%"} {
 # be selected from a file picker first.
 set vivadoDefaultBitstreamFile [ get_property DIRECTORY [current_run] ]/[ get_property top [current_fileset] ].bit
 file copy -force $vivadoDefaultBitstreamFile [pwd]/[current_project].bit
+
+# Copy the .bin file too
+set vivadoDefaultBitstreamBinFile [ get_property DIRECTORY [current_run] ]/[ get_property top [current_fileset] ].bin
+file copy -force $vivadoDefaultBitstreamBinFile [pwd]/[current_project].bin
```

Comment:
That copying is only for backwards-compat. I wouldn't do that for newly introduced functionality.

Reply:
Ok, will remove